mesh/mesh.hpp

```diff
@@ -652,12 +702,7 @@ class Mesh
        reorders vertices, edges and faces along with the elements. */
    void ReorderElements(const Array<int> &ordering, bool reorder_vertices = true);
 
-   /** Creates mesh for the parallelepiped [0,sx]x[0,sy]x[0,sz], divided into
-       nx*ny*nz hexahedra if type=HEXAHEDRON or into 6*nx*ny*nz tetrahedrons if
-       type=TETRAHEDRON. If sfc_ordering = true (default), elements are ordered
-       along a space-filling curve, instead of row by row and layer by layer.
-       The parameter @a generate_edges is ignored (for now, it is kept for
-       backward compatibility). */
+   /// See @a MakeCartesian3D.
    Mesh(int nx, int ny, int nz, Element::Type type, bool generate_edges = false,
```

Comment:
Making them private might be too breaking, but marking them as deprecated could be a good compromise.

They are used somewhat extensively in the unit tests, so it will require changing a lot of code. Perhaps it's better to do it in a separate PR.

Reply:
I implemented these changes in #2054.